README.md

```diff
@@ -1 +1,40 @@
-Hello!
+# The Postgres Standard Library
+
+`Postgrestd` is a custom variant to the Rust Standard Library. This is the main source code repository for `postgrestd`. 
+
+## Key Features
+
+The goal of the library is to prevent code from accessing 
+
+  * file handle operations
+  * internals of the postgres process 
+  * the OS with the permissions of the server process
+  
+The library exposes certain calls into the environment, in the pattern of libc, instead of directly interfacing with the C standard library, the `postgrestd` equivalent functions are called. When it is not applicable we deny the code access, avoiding errors. Code that relies on having access to the environment will panic and abort the transaction.  
+
+The key to managing this is replacing the allocator so that all memory allocations are inside the Postgres context, ensuring that data is torn down properly.
+
+
+## Get Started
+
```

Comment:
I know this is blank for a reason but I am starting to wonder exactly _what_ should go in here? Setup instructions for development? Examples of how it's used?

Reply:
It depends on the focus of the readme.  I am guessing that if there are users of this library they are developers looking to build plrust based things for themselves.  I was thinking this was the here is how you get this setup to build, but ... it could alos have the same focus as pgx (the extension writer) rather than the postgrestd developer.

Reply:
```suggestion

`postgrestd` is intended to be used as part of [PL/Rust](https://github.com/tcdi/plrust).
```